### PR TITLE
Remove admin

### PR DESCRIPTION
**Pull request by meker12**
Closing out an old issue

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-a608d5b4c9/docs/cloud-identity/v2/)

File: api-docs/general-api-info/request-response-types.rst [hunk 1]
```diff
@@ -131,7 +131,7 @@ XML format.
          </token>
          <user name="demoauthor" rax-auth:defaultRegion="DFW" rax-auth:federated="false">
               <roles>
-                   <role id="1" name="identity:admin" description="Admin Role."/>
+                   <role id="3" name="user:admin" description="User Admin Role."/>
```
Comment (kidster) [inline api-docs/general-api-info/request-response-types.rst]: Name would be "identity:user-admin"

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-bc276f2725/docs/cloud-identity/v2/)

**Comment by meker12** [comment]
That was the big one user call. The one that users can use is add user. I had to name them differently.

**Comment by catlook** [comment]
Closes issue #60

**Pull request merged**